token/js/examples/create_mint_and_transfer_tokens.js

```diff
@@ -0,0 +1,77 @@
+import * as web3 from '@solana/web3.js';
+import * as splToken from '@solana/spl-token';
+
+(async () => {
+  // Connect to cluster
+  var connection = new web3.Connection(
+    web3.clusterApiUrl('devnet'),
+    'confirmed',
+  );
+
+  // Generate a new wallet keypair and airdrop SOL
+  var fromWallet = web3.Keypair.generate();
+  var fromWalletAirdropSignature = await connection.requestAirdrop(
+    fromWallet.publicKey,
+    web3.LAMPORTS_PER_SOL,
+  );
+  //wait for airdrop confirmation
+  await connection.confirmTransaction(fromWalletAirdropSignature);
+
+  // Generate a new random public key and airdrop 1 SOL
+  var toWallet = web3.Keypair.generate();
+  var toWalletAirdropSignature = await connection.requestAirdrop(
+    toWallet.publicKey,
+    web3.LAMPORTS_PER_SOL,
+  );
+  //wait for airdrop confirmation
+  await connection.confirmTransaction(toWalletAirdropSignature);
```

Comment:
This isn't necessary because this example never uses the toWallet to pay for a transaction.